landscape/prod/maps/sites.map

```diff
@@ -252,7 +252,13 @@ _/auctions content ;
 _/auth-test content ; #top-level for authentication load test
 _/aux content ;
 _/av content ;
+_/av/abroad wordpress ;
 _/av/cader wordpress ;
```

Comment:
Should there be an entry for `av/celop` in here? It's present in the wp-admin redirects page but is missing here.

Reply:
There should not have been this morning because they were delayed, but luckily it took me so long to get back to this that there now should be an av/celop _and_ an av/asllrp, both of which I will now add.